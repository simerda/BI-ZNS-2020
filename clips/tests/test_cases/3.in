ano
2
ne
ne
ano
ano
