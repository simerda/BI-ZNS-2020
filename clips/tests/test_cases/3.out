CLIPS> Mas figurku ve hre (ano/ne)?
Co jsi naposled hodil (1/2/3/4/5/6)?
Mas nejakou figurku, ktere zbyva N poli do domecku? (ano / ne)
Mas nejakou figurku, ktera ma N poli pred sebou cizi figurku? (ano / ne)
Máš nějakou figurku, která má 6 a méně polí za sebou cizí figurku?
Je jich vice nez 1? (ano / ne)
Tahni tou, ktera je nejdal od startu.

